app/models/week_day.rb

```diff
@@ -9,6 +9,8 @@
 #  active       :boolean          default("false")
 #  timestamps   :string
 #  store_id     :bigint
+#  created_at   :datetime         default("2020-05-05 20:33:42.790398"), not null
```

Comment:
why does this have a default? shouldn't this be handled by rails automatically?

Reply:
This may be the `annotate` gem derping, and evaluating a default to `NOW()`.

Reply:
Nevermind my comment. The migration is wrong:

~~~~ ruby
# db/migrate/20200505203155_add_timestamps_to_week_days.rb
class AddTimestampsToWeekDays < ActiveRecord::Migration[6.0]
  def change
    add_column :week_days, :created_at, :datetime, null: false, default: Time.now
    add_column :week_days, :updated_at, :datetime, null: false, default: Time.now
  end
end
~~~~

Reply:
This was added just because jsonapi-resources can't handle models without `created_at` and `updated_at` (or maybe it can if we change the cache key, but I didn't test it).
But since we don't use it anymore, we can create a migration to move it.